tensorflow/core/kernels/adjust_hsv_op_common.h

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include "tensorflow/core/framework/op_kernel.h"
+#include "tensorflow/core/framework/tensor.h"
+
+#ifndef TENSORFLOW_CORE_KERNELS_ADJUST_HSV_OP_COMMON_H_
+#define TENSORFLOW_CORE_KERNELS_ADJUST_HSV_OP_COMMON_H_
+
+namespace tensorflow {
+
+class AdjustHueOpBase : public OpKernel {
+
+protected:
```

Comment:
The indentation within this class is lost. Please refer to the style guide as a reference. 

https://google.github.io/styleguide/cppguide.html#Class_Format